src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -599,7 +599,23 @@ public override bool Equals(object obj)
             return (object)this == obj;
         }
 
-        public bool Equals(ISymbol other)
+        bool IEquatable<ISymbol>.Equals(ISymbol other)
```

Comment:
This is a good change -- we can't get rid of all internal usages yet, but this should help catch a lot.